.env

```
@@ -326,7 +326,7 @@ VITE_FEATURE_HYPEREVM=true
 VITE_FEATURE_MANTLE=false
 VITE_FEATURE_INK=false
 VITE_FEATURE_CRONOS=false
-VITE_FEATURE_MEGAETH=false
+VITE_FEATURE_MEGAETH=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FEATURE_MEGAETH key should go before the VITE_FEATURE_MONAD key

(UnorderedKey)